xrdp/xrdp_mm.c

```diff
@@ -1023,138 +1034,21 @@ dynamic_monitor_data_first(intptr_t id, int chan_id, char *data, int bytes,
-    error = xrdp_wm_load_static_colors_plus(wm, 0);
-    if (error != 0)
-    {
-        LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                  " xrdp_wm_load_static_colors_plus failed %d", error);
-        return error;
-    }
-
-    error = xrdp_wm_load_static_pointers(wm);
-    if (error != 0)
-    {
-        LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                  " xrdp_wm_load_static_pointers failed %d", error);
-        return error;
-    }
-    /* resize the main window */
-    error = xrdp_bitmap_resize(wm->screen,
-                               description->session_width,
-                               description->session_height);
-    if (error != 0)
-    {
-        LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                  " xrdp_bitmap_resize failed %d", error);
-        return error;
-    }
-    /* redraw */
-    error = xrdp_bitmap_invalidate(wm->screen, 0);
-    if (error != 0)
-    {
-        LOG_DEVEL(LOG_LEVEL_ERROR,
-                  "process_dynamic_monitor_description:"
-                  " xrdp_bitmap_invalidate failed %d", error);
-        return error;
-    }
-
-    if (module != 0)
-    {
-        error = module->mod_server_version_message(module);
-        if (error != 0)
-        {
-            LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                      " mod_server_version_message failed %d", error);
-            return error;
-        }
-        error = module->mod_server_monitor_resize(
-                    module,
-                    description->session_width,
-                    description->session_height);
-        if (error != 0)
-        {
-            LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                      "mod_server_monitor_resize failed %d", error);
-            return error;
-        }
-        error = module->mod_server_monitor_full_invalidate(
-                    module,
-                    description->session_width,
-                    description->session_height);
-        if (error != 0)
-        {
-            LOG_DEVEL(LOG_LEVEL_ERROR, "process_dynamic_monitor_description:"
-                      "mod_server_monitor_full_invalidate failed %d", error);
-            return error;
-        }
-    }
-
-    wm->client_info->display_sizes.monitorCount = description->monitorCount;
-    wm->client_info->display_sizes.session_width = description->session_width;
-    wm->client_info->display_sizes.session_height = description->session_height;
-    g_memcpy(wm->client_info->display_sizes.minfo,
-             description->minfo,
-             sizeof(struct monitor_info)
-             * CLIENT_MONITOR_DATA_MAXIMUM_MONITORS);
-    g_memcpy(wm->client_info->display_sizes.minfo_wm,
-             description->minfo_wm,
-             sizeof(struct monitor_info)
-             * CLIENT_MONITOR_DATA_MAXIMUM_MONITORS);
+    LOG(LOG_LEVEL_INFO,
```

Comment:
LOG_DEVEL() ? This doesn't look like user-specific logging.

Reply:
Fixed.